### PR TITLE
1 ton outfit: Small Radiator

### PR DESCRIPTION
**Pull request by retrue**
This is my first pull request ever.
It is a tiny cooling device with an outfit size of 1. I have tried to make it useful but expensive so it cannot be abused. I think the price is well balanced. To be sold in the Deep Sky Advanced outfitters.
To the sprite I used as base the tiny battery and the tiny battery hai and edited them in Gimp. Of course, itshould be changed for another sprite made by somebody with skills.
The boss is free to change this commit in any way he considers or closed it if he doesn't like it. But I would be glad if it was accepted with only the sprite changed.

EDIT. I have made so many changes in the last minutes after reading suggestions, that I will explain them here.
From 1 mass, 1.5 cooling and 7,500 price that was overpowered, I have changed it to 1 mass, 0,9 cooling and 20,000 price. It costs double that cooling ducts that have 5 mass and 5 cooling.

EDIT 2. Added the Small Radiator to the Lovelace Advanced outfitters too.

### CODE REVIEW

**Comment by Amazinite** [comment]
WAAAAAYYYYYYY too powerful.
Smaller outfits should be less efficient, but this is more efficient than even the Liquid Nitrogen Cooler.
A far better cooling number would be 0.5.

**Comment by Bladewood** [comment]
As it is, the new radiator has a cooling:mass ratio of 1.5:1. For comparison, regular cooling ducts have a 1:1 ratio. This is, without a doubt, overpowered.
0.5 would work, but I feel like that is too weak. Maybe 0.75?

**Comment by Amazinite** [comment]
Looking at the Supercapacitor compared to the LP036a Bat Pack, 75 energy/mass at 2 mass vs 100 energy/mass at 10 mass, that 0.75 cooling number would fit as well, but the cost would need to go up.

**Comment by Amazinite** [comment]
The cost should probably be 20k+, not 10k.

**Comment by retrue** [comment]
Okay, I have changed cooling to 0.75 and price to 10,000. But I would like to increase cooling to 1.

**Comment by Amazinite** [comment]
If you set the cooling to 1 and keep the mass at 1 then there's no point in it existing, since it would then have the same cooling/mass as the cooling ducts. Being 1/5th the size, it would make the cooling ducts obsolete. If it's 1 mass for 1 cooling/frame then it would have to be far more expensive than 10k, maybe upwards 50k, which is nearly at the level of the Liquid Nitrogen Cooler.

**Comment by Bladewood** [comment]
That simply isn't a feasible option. Given the fact that it only occupies one outfit space, we need to somehow make it inferior to a full-scale cooling duct.

**Comment by retrue** [comment]
At Amazinite. Cooling ducts wouldn't be obsolete, because to get the same cooling you would need to expend 50k instead of the 10k that cooling ducts costs.

**Comment by Amazinite** [comment]
Smaller outfits = less efficient.
More modular outfits (fitting into 1, 2, 3 outfit space) = more expensive.

Sitting at 1 mass and 1 cooling, you're looking at a price tag of ~22k each.
At 1 mass and 0.75 cooling, ~16k each is looking better.
At 1 mass and 0.5 cooling, ~11k each would be the price.

**Comment by Bladewood** [comment]
Try 0.9?

**Comment by Amazinite** [comment]
Now... 2x image at the ready?

**Comment by retrue** [comment]
Okay, changed to 0.9 cooling and 20k credits. And yes, I created first the 2x image.
smallradiator double

**Comment by KiLEdEnNis** [comment]
I still don't feel a 1 mass cooling is relevant
a supercapacitor is because if you need that little bit of battery for launchers you can get it for 2 space, a tiny cooling may only make a difference in an interceptor, a tiny difference, and too expensive to use it at that stage

**Comment by retrue** [comment]
The idea was to create a 1 ton outfit that were a bit useful to, as I say in the description of the outfit, to use all the available outfit space in a ship. I hate it when it is left with 1 free outfit space (a cargo scanner is only useful in the flagship and the new control units from the robotic Kor are totally useless).

**Comment by pcordes** [comment]
"Update filename" is not a useful commit message: it should be a summary of what the change does.  Look at other commit titles / messages in the project history to see what kind of titles are appropriate.  You can google "useful commit messages" for articles on the topic from many projects.

Related changes to different files should all be part of the same git commit.  git isn't CVS.  In this case it doesn't actually break anything to add the outfit without having it for sale anywhere, but it's not very useful and both changes are small, so I would have put both changes into one commit.

**Comment by pcordes** [comment]
Being able to pack your ship all the way to max outfit capacity sounds nice, but may be too good.  It takes away some of the challenge of optimizing your loadout.  Although really that's more frustrating than fun. A couple tons of inefficient cooling is maybe an interesting option in some cases.  I mean, I've had ships where LN + water (30 tons) of cooling wasn't quite enough, but LHe (40 tons) was way more than I needed.

**Comment by retrue** [comment]
@pcordes I am a complete n00b so I apologized for the mistakes. There are so many changes because I was listening to the suggestions I received after submitting the PR. I tried my best. In fact, I was even surprised when the PR appeared here. I thought it was going to be more difficult.
I don't understand this: 

> In this case it doesn't actually break anything to add the outfit without having it for sale anywhere.

In this PR the Small Radiator is for sale in 7 systems: advanced outfitters in the Deep and in the Paradise Worlds.

**Comment by LocalGod79** [comment]
this outfit might be better off as a part of the Independent Outfitter proposed in #1664

File: data/outfits.txt [hunk 1]
```diff
@@ -279,6 +279,15 @@ outfit "Fuel Pod"
...
 	description "A fuel pod increases your fuel capacity, allowing your ship to make one additional hyperspace jump before it needs to land and refuel."
+	
+outfit "Small Radiator"
+	category "Systems"
+	cost 20000
+	thumbnail "outfit/small radiator"
+	"mass" 1
+	"outfit space" -1
+	"cooling" 0.9
+	description "A tiny and efficient cooling system whose only shortcoming is its high price. Ideal for rich people who wants to use all the available space in theirs ships. Radiators work by radiating heat energy away as infrared light."
```
Comment (Bladewood) [inline data/outfits.txt]: >A radiator works by radiating heat energy away as infrared light. It is a tiny and efficient cooling system, and is ideal for rich captains who want to use all the available space in their ships.

**Comment by retrue** [comment]
I am closing this pull request. It has been my first one and it is messy. And I made a mess too with my fork and branches and I cannot submit another PR that I have ready.
I will submit both PR later today.
Sorry for the inconvenience.

**Pull request closed without merging**